packages/components/checkbox/src/composables/use-checkbox.ts

```diff
@@ -50,7 +39,51 @@ export const useCheckbox = (
     isLabeledByFormItem,
   })
 
-  setStoreValue(props, { model })
+  const setStoreValue = () => {
+    function addToStore() {
+      if (isArray(model.value) && !model.value.includes(actualValue.value)) {
+        model.value.push(actualValue.value)
+      } else {
+        model.value = props.trueValue || props.trueLabel || true
+      }
+    }
+    props.checked && addToStore()
+  }
+
+  setStoreValue()
+
+  useDeprecated(
+    {
+      from: 'label act as value',
+      replacement: 'value',
+      version: '3.0.0',
+      scope: 'el-checkbox',
+      ref: 'https://element-plus.org/en-US/component/checkbox.html',
+    },
+    computed(() => isGroup.value && isNil(props.value))
```

Comment:
Maybe we should just use the encapsulated method.
```suggestion
    computed(() => isGroup.value && isPropAbsent(props.value))
```
https://github.com/element-plus/element-plus/blob/4fa54dde2dd99a087e6a0cd579cc137ea581ae53/packages/utils/types.ts#L29-L31

Reply:
Yes, I'll change it.